### PR TITLE
advancecomp: Add aliases for enhanced findability

### PR DESCRIPTION
**Pull request by mat**
advancecomp: Add aliases for enhanced findability

### CODE REVIEW

File: Library/Formula/advancecomp.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Advancecomp < Formula
-  desc "Recompression utilities for .PNG, .MNG, .ZIP, and .GZ files"
+  desc "Recompression utilities for .PNG, .MNG, .ZIP, and .GZ files (advzip, advpng, advmng and advdef)"
```
Comment (mistydemeo) [inline Library/Formula/advancecomp.rb]: The `desc` field is limited to 80 characters; this is a bit too long. As an alternative, maybe creating aliases under those names would make this more discoverable?

**Comment by bfontaine** [comment]
Note that if you use [`homebrew-command-not-found`](https://github.com/Homebrew/homebrew-command-not-found) the formula is suggested for any of the binaries:
```shellSession
$ advzip
The program 'advzip' is currently not installed. You can install it by typing:
  brew install advancecomp
```

**Comment by mat** [comment]
@mistydemeo thanks, I didn't know about aliases, that's much better indeed. I changed back the comment and added aliases instead.

**Comment by mat** [comment]
@bfontaine thank you, wasn't aware of `command-not-found`, I really like it.

However, this is one of the cases where you can only benefit from this tool when you're *aware* of the fact it exists. Any plans to integrate this behaviour into homebrew's core? Or mention/link to it when `home info` and `home search` fail?

**Comment by bfontaine** [comment]
@mat Yes we plan to integrate it.

**Comment by mat** [comment]
@mistydemeo Do you guys think it makes sense to include these aliases? If not I am happy to close this pull request

**Pull request closed without merging**